### PR TITLE
Show owner name in marketplace card for each plugin

### PR DESCRIPTION
**Pull request by AltamashShaikh**
### Description:

Show owner name in marketplace card for each plugin
Fixes: #PG-3773

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

File: matomo.js
Comment (sgiehl) [inline matomo.js:1]: This file should be removed again. Same for `piwik.js`

File: plugins/Marketplace/vue/src/PluginList/PluginList.vue [hunk 1]
```diff
@@ -71,6 +71,12 @@
                 <div class="card-description">{{ plugin.description }}</div>
               </div>
               <div class="card-content-bottom">
+                <div class="owner">
+                  {{translate('Marketplace_CreatedBy')}}
+                  <span
+                    v-if="plugin.owner === 'piwik' || plugin.owner === 'matomo-org'"> Matomo</span>
+                  <span v-else> {{plugin.owner}}</span>
```
Comment (sgiehl) [inline plugins/Marketplace/vue/src/PluginList/PluginList.vue]: feels better readable with spaces
```suggestion
                  <span v-else> {{ plugin.owner }}</span>
```

**Review by sgiehl — CHANGES_REQUESTED** [review]
Left some comments. Otherwise this looks good to me.

File: matomo.js
Comment (AltamashShaikh) [reply matomo.js:1]: My bad, Thanks I have removed both the files

File: plugins/Marketplace/vue/src/PluginList/PluginList.vue (hunk 1 above)
Comment (AltamashShaikh) [reply plugins/Marketplace/vue/src/PluginList/PluginList.vue]: done :+1:

**Review by sgiehl — APPROVED** [review]
Should be good to merge, once the tests passed.

**Pull request merged**